docs/src/domains/steps/tls.md

```diff
@@ -75,11 +78,18 @@ For the new certificate to be taken into account, you need to [redeploy the envi
 platform environment:redeploy
 ```
 
-{{< note theme="info" title="Success!" >}}
+Your site is now live and accessible to the world as soon as the DNS propagates.
```

Comment:
:lady_beetle: That's not what this doc is about. This is about the certificate. It can be live before doing the doc. (Also, DNS doesn't propagate)